src/repository.py

```diff
@@ -37,24 +37,24 @@
 GITHUB_HOSTNAME = "github.com"
 ORIGIN_NAME = "origin"
 HTTPS_URL_PATTERN = re.compile(rf"^https?:\/\/.*@?{GITHUB_HOSTNAME}\/(.+\/.+?)(.git)?$")
-ACTIONS_USER_NAME = "upload-charms-docs-bot"
-ACTIONS_USER_EMAIL = "upload-charms-docs-bot@users.noreply.github.com"
-ACTIONS_PULL_REQUEST_TITLE = "[upload-charm-docs] Migrate charm docs"
+ACTIONS_USER_NAME = "discourse-gatekeeper-docs-bot"
+ACTIONS_USER_EMAIL = "discourse-gatekeeper-bot@users.noreply.github.com"
+ACTIONS_PULL_REQUEST_TITLE = "[discourse-gatekeeper] Migrate charm docs"
 ACTIONS_PULL_REQUEST_BODY = (
-    "This pull request was autogenerated by upload-charm-docs to migrate "
+    "This pull request was autogenerated by discourse-gatekeeper to migrate "
     "existing documentation from server to the git repository."
 )
 PR_LINK_NO_CHANGE = "<not created due to no changes in repository>"
 TAG_MESSAGE = (
-    "tag created by upload-charm-docs to mark the latest push to discourse, managed by "
-    "upload-charm-docs, changes or removal of this tag may lead to unexpected behaviour"
+    "tag created by discourse-gatekeeper to mark the latest push to discourse, managed by "
+    "discourse-gatekeeper, changes or removal of this tag may lead to unexpected behaviour"
 )
 
 CONFIG_USER_SECTION_NAME = "user"
 CONFIG_USER_NAME = (CONFIG_USER_SECTION_NAME, "name")
 CONFIG_USER_EMAIL = (CONFIG_USER_SECTION_NAME, "email")
 
-BRANCH_PREFIX = "upload-charm-docs"
+BRANCH_PREFIX = "discourse-gatekeeper"
```

Comment:
This is also a breaking change for existing migration PRs, should we also support both for a while?